gsr_booking/migrations/0002_auto_20210129_1527.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1.4 on 2021-01-29 20:27
```

Comment:
How did this end up being generated? Looks like the code was updated but someone didn't run makemigrations on it?

Reply:
yea im not entirely sure. I ran makemigrations to populate the database for the user app, but i guess it also made a new migration for the gsr_booking app as well ?